LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -4829,19 +4848,19 @@ LocHelpText="Grant a movement action to an ally."
 LocPromotionPopupText="<Bullet/> Rescue Protocol can only be used on allies with no action points left.<br/><Bullet/> It cannot be used on concealed, suppressing or overwatching units.<br/><Bullet/>An ally cannot benefit from both Command and Rescue Protocol in the same turn.<br/><Bullet/> Uses one action.<br/><Bullet/> Charges increase with improved Gremlin technology.<br/>
 
 [AssaultRifle_BM X2WeaponTemplate]
-TacticalText="<Bullet/> The Plasma Rifle is a general purpose, supremely powerful primary infantry weapons system.\n<Bullet/> Firing the primary weapon always ends the soldier's turn.\n<Bullet/> Use upgrades looted from enemies to improve the Plasma Rifle's tactical effectiveness.<br/><Bullet/> Plasma weapons also cause targets to take 1 additional damage on future hits."
+TacticalText="<Bullet/> Plasma weapons cause targets to take 1 additional damage on future hits.\n<Bullet/> The Plasma Rifle is a general purpose, supremely powerful primary infantry weapons system.\n<Bullet/> Firing the primary weapon always ends the soldier's turn.\n<Bullet/> Use upgrades looted from enemies to improve the Plasma Rifle's tactical effectiveness.<br/>"
```

Comment:
Does this apply if the future hit is from a non-plasma weapon, like a coilgun?

Reply:
This was the LW2 localization for the rupture on all guns, i just moved it to the top in each entry. I do agree that its a bit unclear though. Perhaps we can directly say that it applies rupture instead.

Reply:
I can't remember how it works. Does the +1 Rupture effect apply to non-beam weapons too?

Reply:
Yes, its universal.

Reply:
OK, thanks. Then I'm happy with the text since that's what it implies (to me at least)

Reply:
Alright, you can merge then, i added the other changes you requested in a commit.